packages/firestore/src/remote/watch_change.ts

```diff
@@ -188,30 +192,84 @@ export class WatchChangeAggregator {
    * so to know if a target is active, there must be no pending acks we're
    * waiting for and it must be in the current list of targets that the client
    * cares about.
```

Comment:
This comment makes more sense `isActiveTarget`. We should probably combine it with the comment you just added (that describes that `isActiveTarget` calls this method).

Once that is done, I don't think `queryDataForActiveTarget` needs an actual comment. It's pretty self-explanatory.

Reply:
I think I disagree. This is the method that is actually determining whether or not a target is 'active'. This is confusing in the presence of `isActiveTarget`, hence the comment for that method that it defers to this one.

Reply:
The comment as is doesn't describe what this function does, but rather describes what `isActiveTarget` is used for. If you intend to keep this comment, can you emphasize that it returns QueryData? You can use the current comment to explain what an "active target" is. 

How about something along the lines of:

```
Returns the QueryData for a target that has been acknowledged by the backend and is still subscribed to by the user. This method is used to ensure that only events for these active targets are processed.
```

Please rewrite this as you please, as I am currently suffering from writer's block.

Reply:
Rewritten.